Add debugDir parameter to dynamic-simulation, dynamic-security-analysis, shortcircuit and ampl-executor (#3455)
powsybl/powsybl-core: docs/simulation/shortcircuit/parameters.md

]
````

**detailedReport**

> This property indicates if the computation should be done with detailed reporting. If `true`, detailed reports are returned.
> If `false`, synthesized reports are returned.

Reviewer: ```suggestion
This property indicates whether the computation should produce detailed reporting. If `true`, detailed reports are returned; if `false`, synthesized (summary) reports are returned.
```

Author: done, I put "summarized reports"

**debugDir**

> This property indicates a directory path where debug files will be dumped. If `null`, no file is dumped.

Reviewer: ```suggestion
This property specifies the directory path where debug files will be dumped. If `null`, no files will be dumped.
```

Author: done
